packages/ui/src/theme/tokens/components/button.ts

```diff
@@ -196,7 +520,7 @@ export const button: Required<ButtonTokens<'default'>> = {
   link: {
     backgroundColor: { value: 'transparent' },
     borderColor: { value: 'transparent' },
-    borderWidth: { value: '{space.zero.value}' },
+    borderWidth: { value: '{borderWidths.small.value}' },
```

Comment:
Updated this to have a non-zero border width so that this button variation can work in Window's high contrast mode.